kcidb/test_oo.py

```diff
@@ -203,3 +203,169 @@ def test_two_object(self):
             '>checkout#>build#>test#',
             {"test", "build", "checkout", "revision"}, 2
         )
+
+
+class KCIDBMore00ClientTestCase(kcidb.unittest.TestCase):
+    """More test case for oo.Client"""
+
+    def setUp(self):
+        source = kcidb.db.Client("sqlite::memory:")
+        source.init()
+        source.load(
+            {
+                "version": {"major": 4, "minor": 0},
+                "checkouts": [
+                    {
+                        "id": "_:kernelci:5acb9c2a7bc836e"
+                              "9e5172bbcd2311499c5b4e5f1",
```

Comment:
I haven't looked at the data details, we'll just see how the tests go, but you can have minimal IDs, or IDs actually describing what the object is about. You only need them to comply with the schema. E.g. something like `_:valid1`, `_:valid2`, `_:invalid` and so on. That should make the data easier to read and understand.